chore: prepare a bunch of releases (#2749)
tokio-rs/tracing: tracing-subscriber/CHANGELOG.md

# 0.3.18 (October 12, 2023)

### Fixed
- fix new warnings in Rust 1.72.0 (#2700)

> - fix clippy lints (#2673)

Reviewer: i wouldn't include lint fixes in the changelog

> - use fully qualified names in macros for items exported from std prelude (#2621)

Reviewer: i don't think this change is actually part of the subscriber release --- i think it just touched tests?

Author: ah, true—i didn't look closely enough. only saw that it was impacted.

### Added

> - Implement layer::Filter for Option<Filter> (#2407)

Reviewer: ```suggestion
- Implement `layer::Filter` for `Option<Filter>` (#2407)
```

> - Add Chrono implementations for `FormatTime` (#2690)
> - support `NO_COLOR` in `fmt::Layer` (#2647)

Reviewer: ```suggestion
- **fmt**: Add Chrono implementations for `FormatTime` (#2690)
- **fmt**: Support the `NO_COLOR` environment variable for determining whether `fmt::Layer` should emit ANSI color escape codes  (#2647)
```

### Documented

> - subscriber: document `registry` feature requirement for `fmt` feature (#2590)

Reviewer: ```suggestion
- Document `registry` feature requirement for `fmt` feature (#2590)
```

# 0.3.18 (October 12, 2023)

### Added

> - **filter**: Implement `layer::Filter` for `Option<Filter>` [#2407]
> - **fmt**: Add Chrono implementations for `FormatTime` [#2690]
> - **fmt**: Support the `NO_COLOR` environment variable for determining whether `fmt::Layer` should emit ANSI color escape codes  [#2647]
> - **fmt**: make `format::Writer::new()` public [#2680]
> 
> ### Documented
> 
> - Document `registry` feature requirement for `fmt` feature [#2590]

Reviewer: ```suggestion
- **filter**: Implement `layer::Filter` for `Option<Filter>` ([#2407])
- **fmt**: Add Chrono implementations for `FormatTime` ([#2690])
- **fmt**: Support the `NO_COLOR` environment variable for determining whether `fmt::Layer` should emit ANSI color escape codes  ([#2647])
- **fmt**: make `format::Writer::new()` public ([#2680])

### Documented

- Document `registry` feature requirement for `fmt` feature ([#2590])
```
